docs/docs/user/tools/capella/teamforcapella/commit-messages/index.md

```diff
@@ -0,0 +1,49 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Work with TeamForCapella commits and commit messages
+
+!!! Info
+
+    Please [request a persistent Capella session](../../../../sessions/request.md) before you continue
+    with the instructions here.
+
+## Activate commit messages
```

Comment:
I might rename this to "Enable commit message prompt" (I think "activate" might be misunderstood here as activating commits on the model itself, not the user-specific experience)

Reply:
Done